latex_ref_completions.py

```diff
@@ -55,14 +55,29 @@ def find_labels_in_files(root, labels):
         labels.append(command.args)
 
 
+# grab labels from all open buffers too
+def find_labels_in_open_files(views, labels):
+    for view in views:
+        # XXX: Hardcoded to only look for tex files, so that it doesn't try 
+        # looking for label commands in support files
+        if view.file_name() is None or view.file_name().endswith(".tex"):
```

Comment:
This could probably be more sensibly implemented by checking the syntax (something like `view.score_selector('text.tex.latex', 0) > 0` or at least checking for various extensions supported using the `is_tex_file()` function in the `latextools_utils.is_tex_file` module.